### ISSUE TITLE
Need a .htm on news url

### ISSUE THREAD

**Issue by randy7771026** [TheHackerSociety/hpd#11]
Is news in two places?

http://www.houstontx.gov/police/news.htm

**Comment by naren8642** [comment]
Yes, news is also under social. I can remove that, if that is what you are suggesting.
